app/controllers/bus_stops_controller.rb

```diff
@@ -64,6 +64,7 @@ def outdated
 
   def update
     @stop.assign_attributes stop_params
+    @stop.completed? ? @stop.completed_by = current_user : nil
```

Comment:
Two issues with this:

1. This is logically equivalent to `@stop_completed.by = current_user if @stop.completed?`, because your else case does nothing.
2. It doesn't handle if the stop was already completed.